eng/Packages.Data.props

```diff
@@ -193,9 +193,9 @@
     <PackageReference Update="NSubstitute" Version="3.1.0" />
     <PackageReference Update="NUnit" Version="3.12.0" />
     <PackageReference Update="NUnit3TestAdapter" Version="3.13.0" />
-    <PackageReference Update="OpenTelemetry.Extensions.Hosting" Version="0.8.0-beta.1" />
-    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="0.8.0-beta.1" />
-    <PackageReference Update="OpenTelemetry.Instrumentation.Http" Version="0.8.0-beta.1" />
+    <PackageReference Update="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc2" />
+    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc2" />
```

Comment:
how are these used in this repo? Only for examples?

Reply:
Scroll up to line 137, these references are for Test projects only. I was also confused about this :)